packages/components/src/presentation-components/instance-filter-builder/PresentationFilterBuilder.tsx

```diff
@@ -192,3 +193,32 @@ export function PresentationInstanceFilterBuilder(props: PresentationInstanceFil
     />
   );
 }
+
+/**
+ * Creates [InstanceFilterDefinition]($presentation-common) from [[PresentationInstanceFilterInfo]]. Created definition
+ * can be passed to [PresentationManager]($presentation-frontend) through request options in order to filter results.
+ * @beta
+ */
+export async function createInstanceFilterDefinition(info: PresentationInstanceFilterInfo, imodel: IModelConnection): Promise<InstanceFilterDefinition> {
+  if (!info.filter) {
+    assert(info.usedClasses.length > 0);
```

Comment:
Do we need this assert? Maybe just return empty `expression` if there's no filter or classes?